extra/pattern/diamond.js

```diff
@@ -0,0 +1,22 @@
+export default function diamond(node){
+  let string = "";
+    for (let i = 1; i <= node; i++) {
```

Comment:
@arjavjainsuccessive, please mention the valid name instead of i, j, and string.